service/controller/clusterapi/drainer.go

```diff
@@ -18,6 +18,7 @@ import (
 
 	"github.com/giantswarm/aws-operator/client/aws"
 	v29 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v29"
+	v30 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v29"
```

Comment:
```suggestion
	v30 "github.com/giantswarm/aws-operator/service/controller/clusterapi/v30"
```

Reply:
I don't understand why my editor does this sometimes. Makes 0 sense and is so easy to miss.